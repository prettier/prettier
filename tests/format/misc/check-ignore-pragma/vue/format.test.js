runFormatTest(import.meta, ["vue"], { checkIgnorePragma: true });
