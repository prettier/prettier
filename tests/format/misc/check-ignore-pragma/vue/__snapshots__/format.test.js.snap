// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`with-noformat-pragma.vue - {"checkIgnorePragma":true} format 1`] = `
====================================options=====================================
checkIgnorePragma: true
parsers: ["vue"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<!-- @noformat -->
<template>
  <div>This will be pre-compiled</div>
</template>
<script>
const    x      = 1;
</script>
<style src="./my-component.css"></style>

=====================================output=====================================
<!-- @noformat -->
<template>
  <div>This will be pre-compiled</div>
</template>
<script>
const    x      = 1;
</script>
<style src="./my-component.css"></style>

================================================================================
`;

exports[`with-noprettier-pragma.vue - {"checkIgnorePragma":true} format 1`] = `
====================================options=====================================
checkIgnorePragma: true
parsers: ["vue"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<!-- @noprettier -->
<template>
  <div>This will be pre-compiled</div>
</template>
<script>
const    x      = 1;
</script>
<style src="./my-component.css"></style>

=====================================output=====================================
<!-- @noprettier -->
<template>
  <div>This will be pre-compiled</div>
</template>
<script>
const    x      = 1;
</script>
<style src="./my-component.css"></style>

================================================================================
`;

exports[`without-pragma.vue - {"checkIgnorePragma":true} format 1`] = `
====================================options=====================================
checkIgnorePragma: true
parsers: ["vue"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<template>
  <div>This will be pre-compiled</div>
</template>
<script>
const    x      = 1;
</script>
<style src="./my-component.css"></style>

=====================================output=====================================
<template>
  <div>This will be pre-compiled</div>
</template>
<script>
const x = 1;
</script>
<style src="./my-component.css"></style>

================================================================================
`;
